README.md

```diff
@@ -1,80 +1,87 @@
-#kdb-rest-service
+# kdb-rest-service
 
 This is an open source rest service written in java which can be used to connect to an instance of kdb using json. The rest service can provide a single query to run or call a function predefined on the instance of kdb.
 
-##Pre-requisite Configuration 
-#####Properties
+## Pre-requisite Configuration 
+##### Properties
 There is an application.properties file in the resources folder, connect the rest service and the instance of kdb by updating the properties below:
 
-kdb.host=localhost
+    kdb.host=localhost
 
-kdb.port=6007
+    kdb.port=6007
 
-kdb.username=admin
+    kdb.username=admin
 
-kdb.password=admin
+    kdb.password=admin
 
-server.port=8080
+    server.port=8080
 
 
-#####EndPoints
+##### EndPoints
 The kdb-rest-service provides two endpoints:executeFunction and executeQuery. 
 
 The executeFunction provides a means to call a predefined function and pass parameters to the kdb instance. 
 For example this is the format of a request call a function called plus which passes two arguments labelled "xarg" and "yarg" with values 96.3 and 9.7:
 
 e.g.
 {
-"function_name" : "plus",
-"arguments" : {
-              "xarg" : "96.3",
-              "yarg" : "9.7"
+
+    "function_name" : "plus",
+    "arguments" : 
+            {
+                "xarg" : "96.3",
+                "yarg" : "9.7"
              }
 }
 
 The executeQuery provides a means to provide a query to the kdb instance, by default this endpoint is disabled using the property freeform.query.mode.enabled, to enable change the value to true. 
 For example this is the format of a synchronous query request where the user expects a response to be returned:
 
+e.g.
 {
-"type" : "sync",
-"query" : "select from table",
-"response" : true
+
+    "type" : "sync",
+   
+    "query" : "select from table",
+
+    "response" : true
+
 }
 
-#####Certificates
+##### Certificates
 The requests are sent in HTTPS format and to provide this the project has a self-signed certifiate embedded within. It is strongly recommended that you add your own certificate. Updating the certificate will require and update to the following properties in application.properties:
 
-security.require-ssl=true
+    security.require-ssl=true
 
```

Comment:
I'd get rid of all these blank lines between each property, they're not needed inside code blocks (same in earlier section)